Documentation/architecture.rst

```diff
@@ -208,174 +208,241 @@ label source defaults to ``any:`` which will match all labels regardless of
 their source. If a source is provided, the source of the selecting and matching
 labels need to match.
 
-******************
-Address Management
-******************
+*********
+Endpoints
+*********
+
+Cilium makes application containers available on the network by assigning them
+IP addresses. Multiple application containers can share the same IP address, a
+typical example for this model is a Kubernetes Pod_. All application containers
+which share a common address are grouped together in what Cilium refers to as
+an *endpoint*.
+
+Allocating individual IP addresses enables the use of the entire Layer 4 port
+range by each *endpoint*. This essentially allows multiple application
+container running on the same cluster node to all bind to well known ports such
+`80` without causing any conflicts.
+
+The default behaviour of Cilium is to assign both an IPv6 and IPv4 address to
+every *endpoint*. However, this behaviour can be configured to only allocate an
+IPv4 or IPv6 address. If both an IPv6 and IPv4 address are assigned, either
```

Comment:
would be helpful to link to a section in the Docs that describes such configuration